.travis.yml

```diff
@@ -1,6 +1,9 @@
 language: ruby
 rvm:
   - 2.2.3
+before_script:
+  - sed -f _scripts/baseurl.sed _config.yml > _scripts/_config.yml
+  - mv _scripts/_config.yml _config.yml 
```

Comment:
@nkinkade - How do you feel about giving it a separate name? I think writing to a new file called `_config_deploy.yml` makes it a little clearer what's going on. Overwriting a source-controlled file also feels a little dirty to me.